packages/studio/src/store/hotReloadStore.ts

```diff
@@ -16,6 +16,8 @@ export default async function hotReloadStore(payload: StudioHMRPayload) {
     case "modules":
       await syncFileMetadata(studioData, payload.file);
       break;
+    case "layouts":
```

Comment:
should we be updating the `layouts` in `LayoutSlice` here?

Reply:
yep!  TODO added